generate.js

```diff
@@ -1,6 +1,7 @@
 const fs = require('fs')
 const Mustache = require('mustache')
 const orgs = require('./out/data.json')
+const timestamp =  new Date(fs.statSync('./out/data.json').mtime).getTime()
```

Comment:
Remove one of the spaces after the equals sign.

We should get coala working ASAP